### ISSUE TITLE
Adalight, doesn't re-intialise the serial after LED power cycle even when switching lights off/on.

### ISSUE THREAD

**Issue by Felixrising** [psieg/Lightpack#576]
Software 5.11.2.31
TL:DR: Need to restart Prismatik or recongigure the device to get Adalight working again after Adalight (WLED) LED string  is power cycled. 

I have WLED controller powering a LED strip, it receives Adalight serial (or TMP2 also) from Prismatik, however because the WLED controller switches off when the monitor is switched off (when computer puts display to sleep/system suspend), Prismatik should ideally reconnect or reinitiatilise the connection after display sleep/system suspend, etc, however it doesn't unless I totally restart prismatik or reconfigure the device in Prismatik. Small annoyance, I'd like it to "just work".

**Comment by psieg** [comment]
Sorry for the late reply. Prismatik tries to reinitialize the device normally, I know this works with Lightpack devices. Can you turn on logging and check for "Will try to recreate device" messages in the log?
